### PR TITLE
Cta/random walk

### PR DESCRIPTION
**Pull request by coryalini**
Adds brownian or random walk noise onto the wind direction

### CODE REVIEW

File: src/plots.jl [hunk 1]
```diff
@@ -0,0 +1,57 @@
+
+using MeshCat
+using CoordinateTransformations
+using Rotations 
+using Colors
+using GeometryBasics
+using RobotZoo: PlanarRocket
+using Plots
+
+
```
Comment (coryalini) [inline src/plots.jl]: This part isnt ready yet. Ill remove it from PR

File: src/dynamics.jl [hunk 2]
```diff
@@ -75,4 +84,16 @@ function dynamics_jacobians(x,u,dt)
     A = FD.jacobian(_x -> rk4(_x,u,dt),x)
     B = FD.jacobian(_u -> rk4(x,_u,dt),u)
     return A,B
-end
\ No newline at end of file
+end
+
+
+
+function simulate_random_walk_wind_traj(wd,x, move=1)
```
Comment (SchwartzCode) [inline src/dynamics.jl]: nit: x argument unused

File: src/dynamics.jl [hunk 3]
```diff
@@ -14,6 +14,9 @@ k_m = 1.0 #0.1 # [N*m/rpm]
          [-ℓ*k_T 0 ℓ*k_T 0]
          [k_m -k_m k_m -k_m]]
 
+wind = [1,1, 1]*1.0 # wind direction
+wd = 10 # std on wind angle
```
Comment (SchwartzCode) [inline src/dynamics.jl]: Is there any reason in particular you start at 10?

Comment (SchwartzCode) [reply src/dynamics.jl]: Also i think this is the mean instead of the std dev? If that's what the comment indicates

**Review by SchwartzCode — APPROVED** [review]
LGTM, I had a few small comments

File: src/dynamics.jl (hunk 3 above)
Comment (coryalini) [reply src/dynamics.jl]: Sorry that was a copy pasta from the old quadcopter code.   FIxing

**Pull request merged**